File: PRAK204-2510817320004-RahmaSyaritaJaya.py
import math
M_PI = 3.14286

r = float(input())
t = float(input())
print(f"Volume = {M_PI*r*r*t:.2f}\nLuas = {(2*M_PI*r)*(r+t):.2f}\nKeliling = {2*M_PI*r:.2f}")

r, t = map(float, input().split())
print(f"Volume = {M_PI*r*r*t:.2f}\nLuas = {(2*M_PI*r)*(r+t):.2f}\nKeliling = {2*M_PI*r:.2f}")

File: PRAK204-2510817320004-RahmaSyaritaJaya.c
#include <stdio.h>
#define M_PI 3.14286

int main(){
    float r, t;
    scanf("%f", &r);
    scanf("%f", &t);
    printf("Volume = %.2f\nLuas = %.2f\nKeliling = %.2f\n", M_PI*r*r*t, (2*M_PI*r)*(r+t), 2*M_PI*r);
    
    scanf("%f %f", &r, &t);
    printf("Volume = %.2f\nLuas = %.2f\nKeliling = %.2f\n", M_PI*r*r*t, (2*M_PI*r)*(r+t), 2*M_PI*r);
    return 0;
}
